tests/10-verify.spec.js

```diff
@@ -349,11 +349,89 @@ describe('verify API (presentations)', () => {
   });
 });
 
+describe('tests for multiple credentials', async () => {
+
+  const credentialsCount = [5, 25, 50, 100];
+
+  for(const count of credentialsCount) {
+    it('presentation contains multiple credentials', async () => {
+      const challenge = uuid();
+      const {presentation} = await _generatePresentation({
+        challenge,
+        credentialsCount: count,
+      });
+
+      presentation.verifiableCredential.should.be.an('array');
+      presentation.verifiableCredential.should.have.length(count);
+    });
```

Comment:
This test can be removed now as it was just for you to understand what the private `_generatePresentation` helper function was doing.  We only want tests that test the public APIs in vc-js, not testing the test framework (e.g. the test helper `_generatePresentation`).